code/index.html

```diff
@@ -3,28 +3,82 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>DX Wines</title>
+    <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <!-- video or image as a header is cool :) -->
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    <header>
+      <nav class="nav">
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+      </nav>
+      <h1 class="header-title">DX WINES</h1>
+      <div class="hamburger-menu">
+        <div></div>
+        <div></div>
+        <div></div>
       </div>
-
-      <div class="question" id="section2">Section 2</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+    </header>
+    <main>
+      <h2 class="wine-gallery-heading">Our wines</h2>
+      <section class="wine-gallery">
+        <div>
+          <img src="Images/Red-wine.jpg" alt="Red wine">
+        </div>
+        <div>
+          <img src="Images/White-wine.jpg" alt="White wine">
+        </div>
+        <div>
+          <img src="Images/Rose-wine.jpg" alt="Rosé wine">
+        </div>
+        <div>
+          <img src="Images/Sparkling-wine.jpg"alt="Sparkling wine">
+        </div>
+      </section>
+      <!-- Signup form -->
+      <div class="form-container">
+        <form action="https://httpbin.org/anything" method="POST" class="sign-up-form">
+          <h2>Sign up</h2>
+          <label>Name</label>
+          <input type="text" name="name" class="sign-up-info"required>
+          <label>E-mail</label>
+          <input type="email" name="email" class="sign-up-info" required>
+          <label>Telephone</label>
+          <input type="number" name="telephone" class="sign-up-info">
+          <label>Would you like to subscribe to our newsletter?</label>
+          <label><input type="radio" name="subscribe">Yes</label>
+          <label><input type="radio" name="subscribe">No</label>
```

Comment:
The only problem is that your inputs doesn't have `value` attribute and because of that checking both Yes or No answers are saved in form as `subscribe: on`